contracts/Core/BlockManager.sol

```diff
@@ -67,49 +66,21 @@ contract BlockManager is Initializable, ACL, BlockStorage {
         parameters = IParameters(parametersAddress);
     }
 
-    function getBlock(uint256 epoch) external view returns(Structs.Block memory _block) {
+    function getBlock(uint256 epoch) external view returns(Structs.Block memory) {
         return(blocks[epoch]);
     }
 
-    function getBlockMedians(uint256 epoch) external view returns(uint256[] memory _blockMedians) {
```

Comment:
These Methods were redundant (we can get everything from the block directly), so have removed them.
Let me know if we should have them.
Or If we should remove them in separate PR.